TrainingContainer.Dockerfile

```
@@ -2,11 +2,10 @@ FROM python:3.9-slim
 
 WORKDIR /
 
-COPY requirements.txt .
-RUN apt-get update -y && apt-get install -y gcc && apt-get install -y curl && apt-get install -y unzip
-RUN pip install -r requirements.txt
+COPY pyproject.toml poetry.lock ./
```

```suggestion
COPY pyproject.toml ./
```

might not need to copy over `poetry.lock` over to docker